pkg/ledger/volume_agg.go

```diff
@@ -0,0 +1,133 @@
+package ledger
```

Comment:
I think that everything in this file shouldn't be exposed, because everything is used in this package

Reply:
Make sense.